ansible/roles/operator-pipeline/templates/openshift/tasks/publish-bundle.yml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: publish-bundle
+spec:
+  params:
+    - name: cert_project_id
+    - name: container_digest
+    - name: bundle_versions
+  steps:
+    - name: publish-bundle
+      image: registry.access.redhat.com/ubi8-minimal
+      script: |
+        #! /usr/bin/env bash
+        echo "Starting bundle publish process"
```

Comment:
Please add a placeholder in here to set a _pending_ github status for each IIB job when it starts (in addition to when it completes). I'm not sure if you want to embed that in this task or make use of https://hub.tekton.dev/tekton/task/github-set-status so it's up to you for now.

Reply:
I'd prefer to keep it in one task. Tekton task which you shared basically does one call to GitHub API
https://github.com/tektoncd/catalog/blob/main/task/github-set-status/0.2/github-set-status.yaml